lib/services/presets.go

```diff
@@ -75,6 +75,9 @@ func NewPresetEditorRole() types.Role {
 					types.NewRule(types.KindInstaller, RW()),
 					types.NewRule(types.KindInstance, RO()),
 					types.NewRule(types.KindLoginRule, RW()),
+					types.NewRule(types.KindLicense, RO()),
+					types.NewRule(types.KindDownload, RO()),
```

Comment:
Why are these 2 needed?

Reply:
Oops
They shouldn't be here, removing
I think they came from a merge conflict